docs/development/01-contribution.md

```diff
@@ -0,0 +1,59 @@
+---
+id: contribution
+title: Contribution guidelines
+---
+
+As a decentralized network, KILT depends on the support of its community. There are many ways to contribute to KILT Protocol and the products and services built on it. 
+
+If you are interested in contributing but unsure how to begin, start in our [Clan KILT Discord](https://discord.gg/7uyfMXh6AT) channel. The developers active there, which include the team that originally developed KILT, and the wider KILT community can:
+
+- Provide feedback on proposals or ideas
+- Discuss possible use-cases or feature requests
+- Make suggestions for non-technical contributions, including events, writing, or business models
+- Answer questions about the protocol, services and products.
+
+
+## Feature Requests 
+
+A feature request may be used to change KILT and its services by adding, removing or altering features within the scope of KILT Protocol.
+
+A feature request is a meaningful way to contribute from anyone following the guidelines below:
+
+ - Begin a discussion with the community to ensure that they agree that it adds to the KILT protocol in a meaningful way, supporting its goals. 
+ - Open an issue on the corresponding repository
+ - Give your request a clear title
+ - Provide a written outline of the feature request for discussion
+
+After discussion, if the community agrees that the change should be implemented, the proposer may also make a Treasury proposal to support the work. Check out the treasury proposals section.
+
+## Treasury Proposals
+
+A Treasury proposal is a request to receive funds from the Treasury pool. The proposal should begin with a clear title, a written outline of the idea, and a discussion about implementation or deliverables as outlined above for feature requests. The proposal should be for something that changes or adds to KILT in a meaningful way. 
```

Comment:
```suggestion
A Treasury proposal is a request to receive funds from the Treasury pool.
The proposal should begin with a clear title, a written outline of the idea, and a discussion about implementation or deliverables as outlined above for feature requests.
The proposal should be for something that changes or adds to KILT in a meaningful way. 
```

Reply:
I am missing an important piece of information which states that proposals are funding completed contributions and that one should rather go for multiple ones instead of a big proposal. I believe the best place for this statement is in this paragraph, so I added it to my suggestion below.

```suggestion
... adds value to KILT in a meaningful way. 
In general, a Treasury proposal spent occurs after completing all outlined deliverables and not before.
Thus, it is recommended to rather open multiple subsequent milestone-based proposals instead of a large one to fund contributions.
```

Reply:
Yes, I like the addition. I will add change.

Reply:
```suggestion
A Treasury proposal is a request to receive funds from the Treasury pool. 
The proposal should begin with a clear title, a written outline of the idea, and a discussion about implementation or deliverables as outlined above for feature requests. 
The proposal should be for something that changes or adds value to KILT in a meaningful way. 
In general, a Treasury proposal spent occurs after completing all outlined deliverables and not before.
Thus, it is recommended to rather open multiple subsequent milestone-based proposals instead of a large one to fund contributions.
```